service/src/main/java/uk/gov/hmcts/reform/fpl/service/document/ManageDocumentService.java

```diff
@@ -408,6 +429,75 @@ public UUID getSelectedRespondentId(CaseData caseData) {
         return getDynamicListSelectedValue(caseData.getRespondentStatementList(), mapper);
     }
 
+    public <T extends ConfidentialBundle> List<Element<DocumentWithConfidentialAddress>>
+        getDocWithConfidentialAddrFromConfidentialBundleElements(
```

Comment:
Would be good to make this method name a bit more concise I think ... maybe something like "getDocumentsWithConfidentialAddresses"?